packages/runner/injection/main.js

```diff
@@ -16,6 +16,19 @@ if (!Cypress) {
 Cypress in the parent window but it is missing. This should never happen and likely is a bug. Please open an issue.')
 }
 
+/* eslint-disable */
+const originalSetAttribute = Element.prototype.setAttribute
+
+Element.prototype.setAttribute = function (qualifiedName, value) {
```

Comment:
Maybe we should not monkey-patch this at all if the experimental flag is not true.

Reply:
addressed in c360238ed65cdca535ac55d7e1f3eb24e94889e1

Reply:
forgot a flag and removed it in [37cdc0a](https://github.com/cypress-io/cypress/pull/22568/commits/37cdc0aa8c1fb96bd7c77219da36831ab706abec)